WcaOnRails/app/models/competition.rb

```diff
@@ -1908,6 +1908,7 @@ def serializable_hash(options = nil)
       resultsPosted: results_posted?,
       inProgress: in_progress?,
       dateRange: ApplicationController.helpers.wca_date_range(start_date, end_date),
+      dateCount: (is_probably_over? ? (Date.today - end_date) : (start_date - Date.today)).to_i, # Comp starts in X days / Comp ended X days ago. Calculating here because too nasty in ReactJS.
```

Comment:
Does this function / data also get used outside of your specific use-case (this PR)?
I'd actually rather we calculate this on the client. since this value is pretty specific.

Reply:
I wasn't able to find any other uses of this specific value, but yeah doing it on the client is better for the server anyways so I'll try that.